docs/data_types/table/table.md

```diff
@@ -0,0 +1,116 @@
+# 📊 Table
+
+DocArray supports many different modalities including tabular data.
+This section will show you how to load and handle tabular data using DocArray.
+
+## Load CSV table
+
+A common way to store tabular data is via `.csv` files.
+You can easily load such data from a given `.csv` file into a DocumentArray. 
+Let's take a look at the following example file, which includes data about book and their authors and publishing year.
+
+```text
+title,author,year
+Harry Potter and the Philosopher's Stone,J. K. Rowling,1997
+Klara and the sun,Kazuo Ishiguro,2020
+A little life,Hanya Yanagihara,2015
+```
+
+First we have to define the Document schema describing our data.
+```python
+from docarray import BaseDoc
+
+
+class Book(BaseDoc):
+    title: str
+    author: str
+    year: int
+```
+Next, we can load the content of the csv file to a DocList instance of `Book`s.
+```python
+from docarray import DocList
+
+
+docs = DocList[Book].from_csv(file_path='books.csv')
+docs.summary()
+```
+<details>
+    <summary>Output</summary>
+    ``` { .text .no-copy }
+    ╭────── DocList Summary ──────╮
+    │                             │
+    │   Type     DocList[Book]    │
+    │   Length   3                │
+    │                             │
+    ╰─────────────────────────────╯
+    ╭── Document Schema ──╮
+    │                     │
+    │   Book              │
+    │   ├── title: str    │
+    │   ├── author: str   │
+    │   └── year: int     │
+    │                     │
+    ╰─────────────────────╯
+    ```
+</details>
+
+The resulting DocList contains three `Book`s, since each row of the csv file corresponds to one book and is assigned to one `Book` instance.
```

Comment:
We can also handle TSV (tab-separated values) by switchin the dialect in the `from_csv` method. I think it would be nice to mention.